lib/openrtb2_3/bidResponse.js

```diff
@@ -48,12 +77,21 @@ BidResponseBuilder.prototype.build = function() {
         return sb;
       }).bind(this).then(function(_seatbid){
         this._seatbid = _seatbid;
-        resolve(new BidResponse(
-          this._timestamp || new Date().toISOString(),
-          this._status,
-          this._bidderName,
-          this._seatbid
-        ).removeUndefined());
+
+        var bidResponse = new BidResponse(
+              this._timestamp || new Date().toISOString(),
+              this._status,
+              this._bidderName,
+              this._seatbid,
+              this._id
+            ).removeUndefined();
+
+        var message = this.validateBidResponse(bidResponse);
```

Comment:
This should get an array of validation errors. If empty that the validation was successful.